go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/temporalio/features/features v1.0.0
 	github.com/temporalio/features/harness/go v1.0.0
 	github.com/urfave/cli/v2 v2.25.7
-	go.temporal.io/sdk v1.25.2-0.20231121222453-f9d73bfdf7c8
+	go.temporal.io/sdk v1.25.2-0.20231129163633-77626eee3015
```

Comment:
Not the biggest fan of using untagged SDKs in the features repo `main` branch. But I guess we had already done it.

Reply:
Yeah. As we discussed (this is for public record, hello!) I'm going to merge from now to keep usage simple since this is a breaking change that hit both the SDK and the Server, but will do so next time.